consensus/polybft/command/polybftsecrets/params.go

```diff
@@ -0,0 +1,207 @@
+package polybftsecrets
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/0xPolygon/polygon-edge/secrets/helper"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	dataPathFlag = "data-dir"
+	configFlag   = "config"
+	accountFlag  = "account"
+	networkFlag  = "network"
+	numFlag      = "num"
+
+	// maxInitNum is the maximum value for "num" flag
+	maxInitNum = 30
+)
+
+var (
+	errInvalidNum      = fmt.Errorf("num flag value should be between 1 and %d", maxInitNum)
+	errInvalidConfig   = errors.New("invalid secrets configuration")
+	errInvalidParams   = errors.New("no config file or data directory passed in")
+	errUnsupportedType = errors.New("unsupported secrets manager")
+)
+
+type initParams struct {
+	dataPath         string
+	configPath       string
+	generatesAccount bool
+	generatesNetwork bool
+
+	numberOfSecrets int
+}
+
+func (ip *initParams) validateFlags() error {
+	if ip.numberOfSecrets < 1 || ip.numberOfSecrets > maxInitNum {
+		return errInvalidNum
+	}
+
+	if ip.dataPath == "" && ip.configPath == "" {
+		return errInvalidParams
+	}
+
+	return nil
+}
+
+func (ip *initParams) setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&ip.dataPath,
+		dataPathFlag,
+		"",
+		"the directory for the Polygon Edge data if the local FS is used",
+	)
+
+	cmd.Flags().StringVar(
+		&ip.configPath,
+		configFlag,
+		"",
+		"the path to the SecretsManager config file, "+
+			"if omitted, the local FS secrets manager is used",
+	)
+
+	cmd.Flags().IntVar(
+		&ip.numberOfSecrets,
+		numFlag,
+		1,
+		"the flag indicating how many secrets should be created, only for the local FS",
+	)
```

Comment:
Why do we need this flag?

This flag can be substituted with outside tooling, no need to make the codebase more complex for a UX improvement only devs will use